CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## Version 0.4.2-dev (changes since 0.4.2 go here)
 
 ### Features
-
+* Added `skbio.sequence.replace` for replacing items in a `Sequence` with a given character. ([#1222](https://github.com/biocore/scikit-bio/issues/1222))
```

Comment:
use `skbio.sequence.Sequence.replace`

Minor wording change, but I think it'd be clearer to say: *for assigning a character to positions in a `Sequence`*.